application/client/src/app/ui/views/workspace/headers/menu/template.html

```diff
@@ -0,0 +1,26 @@
+<div class="wrapper">
+    <ul class="columns">
+        <li class="column"
+            *ngFor="let header of controller.headers; let i = index;"
+            (click)="ngOnCheckboxClick($event, i)">
```

Comment:
looks like we need `uuid` of each column to correctly bind it between view and controller or static `index` (as property in header description)